apps/google-docs/package.json

```diff
@@ -28,13 +28,13 @@
     "build:functions": "contentful-app-scripts build-functions --ci",
     "build": "rimraf build && npm run build:frontend && npm run build:functions",
     "upload:app-dev": "contentful-app-scripts upload --bundle-dir ./build --organization-id 6xdLsz6lCsk0yPOccSsDK7 --definition-id 653vTnuQw3j5onU1tUoH6t --token $CONTENTFUL_ACCESS_TOKEN",
-    "deploy:oauth-dev": "aws s3 sync ./build ${STATIC_S3_BASE}/apps/google-docs-dev && aws cloudfront create-invalidation --distribution-id $GOOGLE_DOCS_CLOUDFRONT_DIST_ID --paths \"/*\" > /dev/null 2>&1",
-    "deploy:dev": "npm run build && npm run upload:app-dev && npm run deploy:oauth-dev",
+    "deploy:sync-dev": "aws s3 sync ./build ${STATIC_S3_BASE}/apps/google-docs-dev && aws cloudfront create-invalidation --distribution-id $GOOGLE_DOCS_CLOUDFRONT_DIST_ID --paths \"/*\" > /dev/null 2>&1",
```

Comment:
We should set the values for these Env Vars for Circle CI here:
https://github.com/contentful/apps/blob/master/.circleci/config.yml#L120

Reply:
We might even want to rename `STATIC_S3_BASE` to something else because some other apps (google analytics, optimizely, slack) are already using that variable with a different value in Circle CI:
* https://github.com/contentful/apps/blob/master/.circleci/config.yml#L115